package.json

```diff
@@ -6,8 +6,10 @@
     "@testing-library/jest-dom": "^5.11.4",
     "@testing-library/react": "^11.1.0",
     "@testing-library/user-event": "^12.1.10",
+    "prop-types": "^15.7.2",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
+    "react-helmet": "^6.1.0",
```

Comment:
`react-helmet` is great but it kinda died and isn't very compatible with react 16+
Can you use [react-helmet-async](https://www.npmjs.com/package/react-helmet-async) instead?

Reply:
I didn't know that. I updated it, but also maybe @kimrass14's solution is better?

Reply:
No, what you did is great. Thanks!